cinema/models.py

```diff
@@ -1,10 +1,43 @@
 from django.db import models
 
 
+class Actor(models.Model):
+    first_name = models.CharField(max_length=255)
+    last_name = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.first_name} {self.last_name}"
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self):
+        return f"Cinema name:{self.name} Num of rows and seats:" \
+               f" {self.rows}&{self.seats_in_row}"
```

Comment:
use parentheses instead of backslash to wrap the line

Reply:
Done